jpa/spi/src/main/java/org/jipijapa/plugin/spi/PersistenceProviderAdaptor.java

```diff
@@ -94,5 +94,7 @@ public interface PersistenceProviderAdaptor {
     Object beanManagerLifeCycle(BeanManager beanManager);
 
     void markPersistenceUnitAvailable(Object wrapperBeanManagerLifeCycle);
+
+    void addClassFileTransformer(PersistenceUnitMetadata pu);
```

Comment:
Given the default in most other adaptors is a no-op, this could just be a default no-op method:

```java
default void addClassFileTransformer(final PersistenceUnitMetadata pu) {
}
```